projects/fribidi/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Copyright 2016 Google Inc.
```

Comment:
Please update the license to `Copyright 2021 Google LLC`

Reply:
Done